test/e2e/testsuites/testsuites.go

```diff
@@ -1063,23 +1061,16 @@ func (t *TestAzVolumeAttachment) Cleanup() {
 		framework.ExpectNoError(err)
 	}
 
-	// Delete Primary AzVolumeAttachments
+	// Delete All AzVolumeAttachments for t.underlyingVolume
 	err = t.azclient.AzVolumeAttachments(t.namespace).Delete(context.Background(), GetAzVolumeAttachmentName(t.underlyingVolume, t.primaryNodeName), metav1.DeleteOptions{})
```

Comment:
Doesn't this call only delete primary azvolumeattachments? There is a another loop below that deletes the replicas.

Reply:
This comment was intended not for this single line but rather the entire code block following the comment. Since the `TestAzVolumeAttachment` struct no longer holds `replicaNodeNames`, I thought it might be more appropriate to address in the comment deletion of all AzVolumeAttachment objects attached to `t.underlyingVolume` instead.

Reply:
Ah, I see!